packages/gatsby/src/redux/actions/public.js

```diff
@@ -515,6 +515,7 @@ actions.deleteNode = (options: any, plugin: Plugin, args: any) => {
 
 /**
  * Batch delete nodes
+ * @private
```

Comment:
eh? why is there a private action in the `public.js` file?

EDIT: Ahh, it's to remove the need for the blocklist. Maybe add a comment here explaining why?

Reply:
There is actually another `@private` thing in this file. Added a comment so it will perhaps make sense to whoever has to review it.